_data/government.yml

```
@@ -1,14 +1,14 @@
 websites:
     - name: Agenzia delle Entrate
-      url: http://www.agenziaentrate.gov.it/
+      url: https://www.agenziaentrate.gov.it/
       img: agenziaentrate.png
       tfa: No
       twitter: Agenzia_Entrate
       facebook: agenziadelleentrate
       lang: it
 
     - name: Canada Revenue Agency
-      url: http://www.cra-arc.gc.ca
+      url: https://www.canada.ca/en/revenue-agency/cra-canada.html
```

```suggestion
      url: https://www.canada.ca/en/revenue-agency.html
```
The Page you linked leads to the description page for the migration, I think going to the CRA directly would be better?

---

Ah, thanks for the suggestion. I'll implement it right away.